src/modules/iasql_functions/iasql.ts

```diff
@@ -674,7 +674,7 @@ export async function commit(
         if (dryRun) return applyRes;
       }
     } catch (e) {
-      logger.scope({ dbId }).warn(`Something failed applying for modules with changes.\n${e}`);
+      logger.scope({ dbId, e }).warn(`Something failed applying for modules with changes.`);
```

Comment:
why this change?

Reply:
You can't actually see the error, otherwise, and you can't inspect the error object in mezmo if this isn't done.